initrd/bin/kexec-insert-key

```diff
@@ -57,6 +57,9 @@ tpmr extend -ix 4 -ic generic ||
 # Check to continue
 if [ "$unseal_failed" = "y" ]; then
 	confirm_boot="n"
+	if cmp -s "$bootdir/kexec_lukshdr_hash.txt" /tmp/luksDump.txt > /dev/null 2>&1; then
+		echo "Encrypted disk keys(s) have not been changed since sealed in TPM Disk Unlock Key"
```

Comment:
IMO: `keys` instead of `keys(s)` (3x).  I get that the idea is to permit the possibility that there's only one key, but I think `key(s)` is unnecessarily awkward.  (keys(s) has two Ss too, reads like "Hobbitses" to me :sweat_smile: )

I think I'd only really use `(s)` when we're asking the user something and it needs to be abundantly clear that this is the option for one or more "thing(s)" - and there's no better way to rewrite the prompt, etc.  Otherwise it could be unclear what to do for exactly one "thing".

I think the meaning is clear and we're watering down the real point by making it more awkward.  It doesn't agree with `have` as written either, etc.